src/cli/middlewares/base.ts

```diff
@@ -1,4 +1,6 @@
 import {initLogger} from "./logger.js";
+import updateNotifier from "./updateNotifier.js";
```

Comment:
its `ts` not `js` isn't?

Reply:
Actually, it's JS. We're importing the transpiled modules. Please, check the rest of the imports. It's always `.js`